### DISCUSSION TITLE
How to change the maximum number of commands in the recall buffer and what will be the maximum?

### DISCUSSION THREAD

**Discussion by brandy125** [gnudatalanguage/gdl#1752] [Q&A] [answered]
In my Linux environment the maximum commands that I can recall is 200.
How can I increase this number and what will be the maximum?

**Comment by GillesDuvert** [comment, marked as answer]
Normally, setting !EDIT_INPUT=xxx in, e;g., the IDL_STARTUP file should permit xxx lines of history.
